app/_hub/moesif/kong-plugin-moesif/_index.md

```diff
@@ -56,3 +56,34 @@ If you don't see any logs in Moesif, you may need to restart Kong again.
 ```shell
 kong restart
 ```
+
+## Updating Plugin Configuration
+
+[View upgrade steps on Moesif docs](https://www.moesif.com/docs/server-integration/kong-api-gateway/#updating-plugin-configuration?language=kong-api-gateway&utm_medium=docs&utm_campaign=partners&utm_source=kong). 
+
+##  Identifying users
+
+This plugin will automatically identify API users so you can associate a user's API traffic to user data and other app analytics.
+The default algorithm covers most authorization designs and works as follows:
+
+1. If the `config.user_id_header` option is set, read the value from the specified HTTP header key `config.user_id_header`.
+2. Else if Kong defined a value for `x-consumer-custom-id`, `x-consumer-username`, or `x-consumer-id` (in that order), use that value.
+3. Else if an authorization token is present in `config.authorization_header_name`, parse the user id from the token as follows:
+   * If header contains `Bearer`, base64 decode the string and use the value defined by `config.authorization_user_id_field` (by default is `sub`).
+   * If header contains `Basic`, base64 decode the string and use the username portion (before the `:` character).
+
+For advanced configurations, you can define a custom header containing the user id via `config.user_id_header` or override the options `config.authorization_header_name` and `config.authorization_user_id_field`.
+
+## Identifying companies
+
+You can associate API users to companies for tracking account-level usage similar to user-level usage. This can be done either:
```

Comment:
```suggestion
You can associate API users to companies for tracking account-level usage similar to user-level usage. This can be done in one of the following ways, by order of precedence:
```